### PR TITLE
adding admin service client with pubkey registration

### PR DESCRIPTION
**Pull request by jeffgrunewald**
update the cli to talk to the new (in PR) admin proto service defined in the iot_config proto that is intended to collect administrator/management rpcs of the config service itself.

this re-assigns the gateway region params registration rpc to the admin service's client as well as adds functionality for adding and removing admin/network service public keys to the server to allow validating authenticated requests to secure rpcs

### CODE REVIEW

File: src/key_type.rs [hunk 1]
```diff
@@ -1,8 +1,7 @@
 use crate::Result;
 use anyhow::anyhow;
 use helium_proto::services::iot_config::admin_add_key_req_v1::KeyTypeV1 as ProtoKeyType;
-use serde::{de, Deserialize, Deserializer, Serialize};
-use std::{fmt, str::FromStr};
+use std::str::FromStr;
 
 #[derive(clap::ValueEnum, Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
```
Comment (madninja) [inline src/key_type.rs:6]: hmm.. so now the _only_ reason to have the wrapper type is to allow it to be used in the cli? I'd get rid of it, state possible_values in clap and just do it in the cli handler?

Comment (jeffgrunewald) [reply src/key_type.rs:6]: at the moment this isn't really a wrapper type anymore, but i'm having trouble understanding how to do this without defining this enum locally; the clap documentation isn't showing me how to define possible values without building up a command value parser and that doesn't seem to jive with how the rest of the cli is structured (all based on struct annotations and nested (sub)commands. i also don't see a way to define possible values that maps those options to a type i can implement `From` against which I would need to convert the input to the necessary proto field type when i send the message.

i can easily strip this down to the local enum, `From<KeyType> for ProtoKeyType` and `Display for KeyType` but i don't see how i can make it simpler then that and still enforce the type i need to send in the proto

Comment (jeffgrunewald) [reply src/key_type.rs]: [f4dac7e](https://github.com/helium/helium-config-service-cli/pull/37/commits/f4dac7ebd093098aaf79b4d05199d7d8de6bc8b6)

**Comment by jeffgrunewald** [comment]
Duplicated by https://github.com/helium/helium-config-service-cli/pull/38, which also handles the removal/refactor of deleting eui pairs and devaddr ranges from a route.
Closing in favor of that one

**Pull request closed without merging**